src/tests/pokete/test_difficulty.py

```
@@ -0,0 +1,176 @@
+        # tackle weight: 30 * 1.0 = 30
+        # fire_bite weight: 15 * 2.0 = 30
+        # Total: 60. Fire: 30/60 = 50%
+
+        choices = []
+        for _ in range(1000):
+            choices.append(t.get_decision(None, None, prov_player).attack_value.index)
+
+        fire_count = choices.count("fire_bite")
+
+        # Expected: ~50%
+        self.assertGreater(fire_count, 400, f"AI should prioritize fire move (50% expected). Got {fire_count}/1000")
```

Ruff (0.14.14)

Use a regular `assert` instead of unittest-style `assertGreater`

Replace `assertGreater(...)` with `assert ...`

(PT009)